utils.py

```diff
@@ -0,0 +1,67 @@
+import requests # pylint: disable=import-error
+from google.cloud import language # pylint: disable=import-error
+from google.cloud.language import enums, types # pylint: disable=import-error
+
+SPANISH_API_URL = "http://sesat.fdi.ucm.es:8080/servicios/rest/sinonimos/json/"
+KEY_PARTS_OF_SPEECH = ["ADJ", "NOUN", "NUM", "VERB"]
+
+def gcloud_syntax_extraction(text):
+    """
+    Calls the gcloud natural langauge api's analyze syntax
+    Attributes:
+        text: the text which we wish to analyze
+    Returns:
+        the response given by the api call
+    """
+    client = language.LanguageServiceClient()
+    # pylint: disable=no-member
+    document = types.Document(
+        content=text,
+        type=enums.Document.Type.PLAIN_TEXT)
+    response = client.analyze_syntax(document=document)
+
+    return response
+
+def from_gcloud_remove_extra(gcloud_response):
+    """
+    Removes extra information from the gcloud response. This includes everything that is not
+    the language, the words found, the lemma version of said words, the part of speech for said word
+    and any word that is not part of the key parts of speech constants.
+    Attributes:
+        gcloud_response: the response given by the natural language api
+    Returns:
+        An object with the language, the keywords as defined by the design doc,
+        the lemma and part of speech for said keywords
+    """
+    tokens_shortened = []
+    for token in gcloud_response.tokens:
+        # pylint: disable=no-member
+        part_of_speech_tag = enums.PartOfSpeech.Tag(token.part_of_speech.tag).name
+        if part_of_speech_tag in KEY_PARTS_OF_SPEECH:
+            token_data = {'word' : token.text.content, 'lemma' : token.lemma,
+                          'part_of_speech' : part_of_speech_tag}
+            tokens_shortened.append(token_data)
+
+    response = {'lan' : gcloud_response.language, 'tokens' : tokens_shortened}
+
+    return response
+
+def create_synonym_list_esp(word_arr, max_synonyms=5):
```

Comment:
You could probably refactor the default `max_synonyms` value into a constant so that if you ever decide to reference it somewhere else, you don't have to worry about matching the default value everywhere in your code.

Reply:
Hmmmm I think it's a matter of if I'll ever use it which makes me think this is a typical case of "you'll never use it". For now I think this gives me some flexibility and if I ever do need the max synonyms anywhere else I'll have to change it. But I don't know where I could use it